modules/Product.py

```diff
@@ -0,0 +1,13 @@
+class Product:
+    def __init__(self, name, price:float):
+        self.name = name
+        self.price = float(price)
+    
```

Comment:
Everything inside `def main()` should be under the `if __name__ == "__main__"` and not on a different method. You can also leave it as is (without having it inside a comment).